src/v/datalake/coordinator/coordinator_manager.cc

```diff
@@ -10,21 +10,30 @@
 #include "datalake/coordinator/coordinator_manager.h"
 
 #include "cluster/partition_manager.h"
-#include "coordinator.h"
+#include "datalake/coordinator/coordinator.h"
 #include "datalake/coordinator/state_machine.h"
+#include "iceberg/filesystem_catalog.h"
 #include "model/fundamental.h"
 
 #include <seastar/core/shared_ptr.hh>
 
 namespace datalake::coordinator {
+namespace {
+// TODO: make this configurable.
+const ss::sstring base_location = "redpanda";
```

Comment:
nit: the name should be a bit more self descriptive I think

redpanda-iceberg-catalog or somesuch? (the current name sounds a bit like data directory)

Reply:
Done